scripts/util.py

```diff
@@ -51,8 +53,28 @@ def read_source_root_path():
 def read_proof_root_path():
     return read_path_from_stdin("the 'proofs' directory (usually '.')")
 
+def is_litani_installed():
+    print(textwrap.fill("Litani can either be distributed with your " \
+        "repository in the form of a submodule, in which case you will need " \
+        "to provide the desired directory, or it can be installed using: " \
+        "brew install aws/tap/litani"))
+    if not shutil.which("litani"):
+        print(textwrap.fill("Litani is not installed. Either exit this CLI, " \
+            "install Litani using brew, and run the script again or proceed " \
+            "with the option of including Litani as a submodule"))
+        return False
+    litani_version = subprocess.run(
```

Comment:
I don't think there's any need for this part of the dialogue. If litani is installed with brew, then the version number will change anyway. I don't think there's any point in making the user commit to a particular version number.